### PR TITLE
Inline classes

### PR DESCRIPTION
**Pull request by jeffzoch**
https://github.com/JetBrains/kotlin/blob/68fdeaf865f279cb7a75eaec7e9f4e6cd0646707/compiler/testData/cli/jvm/extraHelp.out#L141 //Inline classes

https://github.com/JetBrains/kotlin/blob/1.3.70/compiler/testData/cli/jvm/extraHelp.out#L74 //Result

### CODE REVIEW

File: kotlin/internal/opts.bzl [hunk 1]
```diff
@@ -49,6 +49,27 @@ _KOPTS = {
...
+        type = attr.bool,
+    ),
+    "x_allow_result_return_type": struct(
+        args = dict(
+            default = False,
+            doc = "Enable kotlin.Result as a return type",
+        ),
+        type = attr.bool,
+    ),
+    "x_opt_in": struct(
```
Comment (restingbull) [inline kotlin/internal/opts.bzl:66]: Drop x_opt_in for this pr? It's been removed form compile.

**Review by restingbull — CHANGES_REQUESTED** [review]

File: kotlin/internal/opts.bzl (hunk 1 above)
Comment (jeffzoch) [reply kotlin/internal/opts.bzl]: Sorry - missed that one

Comment (restingbull) [reply kotlin/internal/opts.bzl]: Apologies for the delay.

**Review by restingbull — APPROVED** [review]

**Comment by restingbull** [comment]
Huh. The CI and contributor checks aren't visible/run?

**Pull request merged**